back/src/companies/resolvers/CompanyPrivate.ts

```diff
@@ -32,9 +27,7 @@ const companyPrivateResolvers: CompanyPrivateResolvers = {
 
     const userId = context.user!.id;
     const role = await getUserRole(userId, parent.orgId);
-    // type casting is necessary here as long as we
-    // do not expose READER and DRIVER role in the API
-    return role as UserRole;
+    return role;
```

Comment:
```suggestion
    return role!;
```
pour résoudre le problème de typing